src/Shared/NativeMethodsShared.cs

```diff
@@ -145,6 +145,27 @@ public enum COWAIT_FLAGS : int
             COWAIT_ALERTABLE = 0x00000002
         }
 
+        /// <summary>
+        /// Processor architecture values
+        /// </summary>
+        internal enum ProcessorArchitectures
+        {
+            // Intel 32 bit
+            Architecture_X86,
```

Comment:
Isn't the usual style to omit the prefix here?  It'll always be used as ProcessorArchitectures.X86 and so on.